test/plugin/scenarios/webflux-scenario/config/expectedData.yaml

```diff
@@ -193,6 +193,66 @@ segmentItems:
         parentSpanId: 8, parentTraceSegmentId: not null, parentServiceInstance: not
           null, parentService: not null, traceId: not null}
       skipAnalysis: 'false'
+  - segmentId: not null
+    spans:
+    - operationName: /testcase/webclient/test1
+      operationId: 0
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Http
+      startTime: not null
+      endTime: not null
+      componentId: 67
+      isError: false
+      spanType: Entry
+      peer: ''
+      skipAnalysis: false
+      tags:
+      - {key: url, value: 'http://localhost:18080/testcase/webclient/test1'}
+      - {key: http.method, value: GET}
+      - {key: status_code, value: '200'}
+      refs:
+      - {parentEndpoint: /projectA/testcase, networkAddress: 'localhost:18080', refType: CrossProcess,
+        parentSpanId: 9, parentTraceSegmentId: not null, parentServiceInstance: not null, parentService: not null, traceId: not null}
+  - segmentId: not null
+    spans:
+    - operationName: /testcase/webclient/ping
+      operationId: 0
+      parentSpanId: -1
+      spanId: 0
+      spanLayer: Http
+      startTime: not null
+      endTime: not null
+      componentId: 67
+      isError: false
+      spanType: Entry
+      peer: ''
+      skipAnalysis: false
+      tags:
+      - {key: url, value: 'http://localhost:18080/testcase/webclient/ping'}
+      - {key: http.method, value: GET}
+      - {key: status_code, value: '200'}
+      refs:
+      - {parentEndpoint: /testcase/webclient/ping, networkAddress: 'localhost:18080', refType: CrossProcess,
+        parentSpanId: 0, parentTraceSegmentId: not null, parentServiceInstance: not null, parentService: not null, traceId: not null}
+  - segmentId: not null
+    spans:
+    - operationName: /testcase/webclient/ping
```

Comment:
It should be and `Entry/testcase/webclient/test1` belong to the same segment, right?

Reply:
This is an exit, the entry is the above one. Isn't it?

Reply:
@dengliming I think the issue here, this exit span isn't in the main segment.

Reply:
Segment should be 
- /testcase/webclient/test1
   -  /testcase/webclient/ping

Reply:
Wait a moment, @arugal , this seems a bug of `refType` and test tool? @dengliming Are these two exit spans both in across-thread ref?

Reply:
No, it isn't. @dengliming please help us confirming the case. From my reading of your codes. 

- /testcase/webclient/test1
  - /testcase/webclient/ping

This is required, your expected data doesn't show parent span or ref, there is something missing.

Reply:
Let me check it.

Reply:
I think these two span not in same thread 
```
/testcase/webclient/test1
/testcase/webclient/ping
```
any idea to transfer the parent context?

Reply:
here 
org.springframework.web.reactive.function.client.DefaultWebClient.DefaultRequestBodyUriSpec#exchange
```
@Override
		public Mono<ClientResponse> exchange() {
			ClientRequest request = (this.inserter != null ?
					initRequestBuilder().body(this.inserter).build() :
					initRequestBuilder().build());
			return Mono.defer(() -> exchangeFunction.exchange(request)
					.checkpoint("Request to " + this.httpMethod.name() + " " + this.uri + " [DefaultWebClient]")
					.switchIfEmpty(NO_HTTP_CLIENT_RESPONSE_ERROR));
		}
```
I think missing context reason is here.

Reply:
A bit tricky. seems the same as httpclient.
The right trace should be:
```
-/projectA/testcase(webflux-projectA-scenario)
	-/testcase/annotation/success (http-client)(webflux-projectA-scenario)
```
but “-/testcase/annotation/success (http-client)(webflux-projectA-scenario)” span don‘t show parent.’ don't know if I misunderstood.

Reply:
Looks like `tarceId` hasn't been passed on, `BodyInserterRequestInterceptor` is in a new thread to run?

Reply:
@dengliming If it is across thread, the parent span is not there, check the ref(type==acorss thread).

Reply:
I think it's been a problem before (without webclient). Maybe the problem was the loss of traceId after springWebFlux processed the request. I'm not sure yet. Do you guys have any ideas？

Reply:
> Looks like `tarceId` hasn't been passed on, `BodyInserterRequestInterceptor` is in a new thread to run?

From my local test. It's not. It seems the webflux server stop span early and hasn't passed traceId to webclient side.